vignettes/how-to-update-released-site.Rmd

```diff
@@ -0,0 +1,269 @@
+---
+title: "How to update a released site"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{How to update a released site}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Checklist
+
+If you're already up-to-speed, here's the checklist:
+
+* Update to usethis >= v2.1.5.9000, maybe with
+  `pak::pkg_install("r-lib/usethis")`.
```

Comment:
```suggestion
  `pak::pak("r-lib/usethis")`.
```

Reply:
I do actually make a point to use `pkg_install()` but I have no strong feelings.